packages/cli/src/commands/verify.ts

```diff
@@ -0,0 +1,125 @@
+import type { CommandModule } from "yargs";
+import { verify } from "../verify";
+import { loadConfig } from "@latticexyz/config/node";
+import { World as WorldConfig } from "@latticexyz/world";
+import { resolveWorldConfig } from "@latticexyz/world/internal";
+import { worldToV1 } from "@latticexyz/world/config/v2";
+import { getOutDirectory, getRpcUrl, getSrcDirectory } from "@latticexyz/common/foundry";
+import { getExistingContracts } from "../utils/getExistingContracts";
+import { getContractData } from "../utils/getContractData";
+import { defaultModuleContracts } from "../utils/defaultModuleContracts";
+import { Hex, createWalletClient, http } from "viem";
+import chalk from "chalk";
+
+type Options = {
+  deployerAddress?: string;
+  worldAddress: string;
+  configPath?: string;
+  profile?: string;
+  rpc?: string;
+  rpcBatch?: boolean;
+  srcDir?: string;
+  verifier?: string;
+  verifierUrl?: string;
+  useProxy?: boolean;
+};
```

Comment:
did you decide not to flip this around and derive this from the options?

Reply:
Sorry, derive what from which options?

Reply:
```ts
const verifyOptions = {
  deployerAddress: ...
  worldAddress: ...
  ...
};

type VerifyOptions = InferredOptionTypes<typeof verifyOptions>;
```

Reply:
oh the types! yes I tried totally restructuring things but it made the process exit early. `yargs` is weird. 

I will try just change the types